src/analytics/Properties.tsx

```diff
@@ -1222,6 +1222,44 @@ export interface SwapTimeMetrics {
   quoteToUserConfirmsSwapElapsedTimeInMs: number // The elapsed time since the quote was received until the user confirmed to execute the swap
 }
 
+export interface SwapTxsProperties {
+  maxGasCost: number | undefined // Max gas cost for the swap (approve + swap) in feeCurrency (decimal value)
```

Comment:
I started using the "gas cost" terminology in https://github.com/valora-inc/wallet/pull/4368 which is also mentioned a few times in https://ethereum.org/en/developers/docs/gas
But I think we should use "gas fee" to be more clear maybe? Which is used more commonly in that doc.

What do you think?